website/partners/templates/partners/partner.html

```diff
@@ -38,7 +38,7 @@ <h4 class="mt-4">{% trans "Address" %}</h4>
                     {% if partner.link %}
                         <h4>{% trans "Website" %}</h4>
                         <p>
-                            <a href="{{ partner.link }}" rel="noopener" target="_blank">{{ partner.link }}</a>
+                            <a href="{{ partner.link }}" rel="noopener" target="_blank" class="wrapped-link">{{ partner.link }}</a>
```

Comment:
This is a nitpick but we have bootstrap 5, so this is equivalent without unnecessary CSS:
```suggestion
                            <a href="{{ partner.link }}" rel="noopener" target="_blank" class="text-break">{{ partner.link }}</a>
```